src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/LuceneDocumentConverterBase.cs

```diff
@@ -488,23 +488,26 @@ private IEnumerable<AbstractField> GetOrCreateNumericField(IndexField field, obj
                 numericField = cached.Field;
             }
 
-            double doubleValue;
-            long longValue;
-
-            switch (BlittableNumber.Parse(value, out doubleValue, out longValue))
+            if (!(value is char))
```

Comment:
This is bad. GetOrCreateNumericField should never be invoked with a value that is not a number. There should be no 'safety' if statements inside this method.